architecture-decisions/0005-resource-per-collection.md

```diff
@@ -0,0 +1,35 @@
+# 5. N IIIFResources per N Collections it's a member of
```

Comment:
This looks good, but I found this title confusing. Maybe "Create a IIIFResource for each Collection that a resource is a member of"?

Reply:
👍 done